src/main/java/io/cdap/plugin/servicenow/sink/ServiceNowSinkConfig.java

```diff
@@ -247,5 +247,5 @@ private Schema convertToServiceNowCompatibleDataTypes(Schema providedFieldSchema
 
     return providedFieldSchema;
   }
-
+  
```

Comment:
remove unnecessary whitespace that was added

Reply:
removed